AUTHORS.rst

```diff
@@ -105,6 +105,7 @@ bugs, provided ideas, etc; roughly in order of appearance):
 * Andrew Janke (https://github.com/apjanke)
 * Unamer (https://github.com/unamer)
 * Karl Dahlke (eklhad@gmail.com)
+* Rick Sayre (https://github.com/whorfin)
```

Comment:
You should be on the first list, not the second.

Reply:
oops
fixed